agenthub/self_discover_agent/reasoning_action_parser.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict, List, Union
+
+from opendevin.core.utils import json
+
+from .agent_state_machine import SelfDiscoverState
+from .reasoning_action import RESEASONING_MODULE_LIST, NestedDict
```

Comment:
```suggestion
from .reasoning_action import REASONING_MODULE_LIST, NestedDict
```

Reply:
Thanks for the review. Typo is fixed now.